test/jdk/tools/launcher/modules/illegalaccess/IllegalAccessTest.java

```diff
@@ -24,34 +24,14 @@
 /**
  * @test
  * @requires vm.compMode != "Xcomp"
- * @modules java.base/jdk.internal.misc
- *          java.base/sun.security.x509
- * @library /test/lib modules
- * @build IllegalAccessTest TryAccess
- *        jdk.test.lib.compiler.CompilerUtils
- *        jdk.test.lib.util.JarUtils
- * @build m/*
- * @run testng/othervm/timeout=180 IllegalAccessTest
+ * @library /test/lib
+ * @build IllegalAccessTest
+ * @run testng IllegalAccessTest
  * @summary Basic test for java --illegal-access=$VALUE
  */
 
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.jar.Attributes;
-import java.util.jar.Manifest;
-import java.util.stream.Stream;
-
-import jdk.test.lib.compiler.CompilerUtils;
-import jdk.test.lib.process.ProcessTools;
-import jdk.test.lib.process.OutputAnalyzer;
-import jdk.test.lib.util.JarUtils;
-
-import org.testng.annotations.DataProvider;
-import org.testng.annotations.Test;
-import static org.testng.Assert.*;
+import jdk.test.lib.process.*;
+import org.testng.annotations.*;
 
 /**
  * Basic test of --illegal-access=value to deny or permit access to JDK internals.
```

Comment:
The comment should probably be updated as the test no longer tests denying or permitting access.

Reply:
Fixed.